controllers/controller_suite_test.go

```diff
@@ -21,17 +21,18 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/medik8s/common/pkg/lease"
```

Comment:
I wanted to go with `<package_name>_suite_test.go`, and I forgot the suffix `s` 👍🏻

Reply:
BTW should `nodemaintenance_controller.go` and `nodemaintenance_controller_test` follow the same pattern? Since they both use "controller" and not "controller**s**"

Reply:
but the package can contain multiple

Reply:
>but the package can contain multiple

Correct, as SNR has multiple controllers

Reply:
filename still is wrong

Reply:
done